hydroserving/core/monitoring_configuration/parser.py

```diff
@@ -0,0 +1,9 @@
+from hydroserving.core.monitoring_configuration.monitoring_configuration import MonitoringConfiguration
+
+
+def parse_monitoring_configuration_selector(in_dict: dict):
+    if in_dict is None:
+        return None
+    return MonitoringConfiguration(
+        batch_size=in_dict.get("batch-size")
```

Comment:
what if `in_dict` doesn't have `batch-size`?

Reply:
You mean that there should be a default value?

Reply:
```
if "batch-size" not in in_dict:
     return None

... = in_dict["batch-size"]
```

Reply:
Added